Apps/WebClient/src/ClientApp/src/models/phsaLoadState.ts

```diff
@@ -0,0 +1,3 @@
+export default interface PhsaLoadState {
```

Comment:
kinda nit, it could be in the same file as immunizationResult.ts

Reply:
moved it to the same file